### PR TITLE
Event page for animal-genomics april meeting

### PR DESCRIPTION
**Pull request by bjlang**
@netlify /events/2024/SIG_animalgenomics_inauguration

### CODE REVIEW

**Comment by bjlang** [comment]
@nf-core-bot fix linting

File: src/content/events/2024/SIG_animalgenomics_inauguration.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: 'animal-genomics: inauguration meeting'
```
Comment (JoseEspinosa) [inline src/content/events/2024/SIG_animalgenomics_inauguration.md:2]: Does not look weird no capitalization?

File: src/content/events/2024/SIG_animalgenomics_inauguration.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+subtitle: Christa Kuehn, Friedrich Loeffler Institute
+type: talk
+startDate: '2024-04-17'
+startTime: '16:00+02:00'
+endDate: '2024-04-17'
+endTime: '17:00+02:00'
```
Comment (JoseEspinosa) [inline src/content/events/2024/SIG_animalgenomics_inauguration.md:8]: Is not 1 hour and a half?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Some questions

**Review by JoseEspinosa — APPROVED** [review]
Thanks! LGTM 🚀

**Pull request merged**